on-prem-installers/onprem/onprem_upgrade.sh

```diff
@@ -782,4 +782,8 @@ done
 # Run after upgrade script
 ./after_upgrade_restart.sh
 
+# Delete platform-keycloak-keycloak-config-cli Job
+kubectl get job platform-keycloak-keycloak-config-cli -n orch-platform -o jsonpath='{.status.conditions[?(@.type=="Failed")].status}' | grep -q "True" && kubectl delete job platform-keycloak-keycloak-config-cli -n orch-platform || echo "Job is not in failed state"
```

Comment:
This should not be needed, it will run and it will be removed by the orchestrator, no need to delete the job